app/components/Collaborators.js

```diff
@@ -20,7 +20,9 @@ type Props = {
 @observer
 class Collaborators extends React.Component<Props> {
   componentDidMount() {
-    this.props.views.fetchPage({ documentId: this.props.document.id });
+    if (!this.props.document.deletedAt) {
```

Comment:
```suggestion
    if (!this.props.document.isDeleted) {
```

Lots of pre-computed attributes available on the model :) 

https://github.com/outline/outline/blob/develop/app/models/Document.js#L95